app.rb

```diff
@@ -4,8 +4,10 @@
 require_relative 'rental'
 require_relative 'book'
 require_relative 'classroom'
+require_relative 'display'
 
 class App
```

Comment:
- [ ] Please, from these methods, kindly move these 2 indicated statements to your `create_person` method:

```RUBY
def create_student
    print 'Age: '
    age = gets.chomp.to_i

    print 'Name: '
    name = gets.chomp

    print 'Has parent permission? [Y/N]: '
    parent_permission = gets.chomp.downcase
    parent_permission = parent_permission == 'y'
    student = Student.new(@class, age, parent_permission, name) # THIS ONE (1)
    @people << student

    puts 'Student created successfully'
    sleep 0.75
  end

  def create_teacher
    print 'Age: '
    age = gets.chomp.to_i

    print 'Name: '
    name = gets.chomp

    print 'Specialization: '
    specialization = gets.chomp

    teacher = Teacher.new(specialization, age, name) # ...AND THIS ONE! (2)
    @people << teacher

    puts 'Teacher created successfully'
    sleep 0.75
  end
```

That way, your create_person method will create all of them 💪

Reply:
Wow, that's a very useful tip. Thank you very much. I appreciate it.

Reply:
@guerreiropedr0 sorry, I was not clear enough on my snippets.  Only the line that creates a new teacher and the line that creates a new student, should be removed from those methods and included on `create_person`.

This one:

```ruby
student = Student.new(@class, age, parent_permission, name)
```

... and this one:

```ruby
teacher = Teacher.new(specialization, age, name)
```
I hope it's clearer now and makes sense to you.

Reply:
![image](https://user-images.githubusercontent.com/57421823/162815501-f02a882f-3d22-46e7-ac68-dde8c5f68ff0.png)

I'm talking about those lines, not the whole methods.

Reply:
Oh, OK, I see it now. It's OK, you don't have to apologize. I think it's my fault that I didn't understand it right away. So thank you so much for explaining it so well.